stat/distuv/logistic_test.go

```diff
@@ -0,0 +1,174 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/floats/scalar"
+)
+
+func TestLogisticParameters(t *testing.T) {
+	t.Parallel()
+
+	var want float64
+
+	l := Logistic{Mu: 1, S: 0}
+
+	want = 2
+	if result := l.NumParameters(); result != int(want) {
+		t.Errorf("Wrong number of parameters: %d != %.0f", result, want)
+	}
+
+	want = 6.0 / 5.0
+	if result := l.ExKurtosis(); result != want {
+		t.Errorf("Wrong excess kurtosis: %f != %f", result, want)
+	}
+
+	want = 0.0
+	if result := l.Skewness(); result != want {
+		t.Errorf("Wrong skewness: %f != %f", result, want)
+	}
+
+	want = l.Mu
+	if result := l.Mean(); result != want {
+		t.Errorf("Wrong mean value: %f != %f", result, want)
+	}
+
+	want = l.Mu
+	if result := l.Median(); result != want {
+		t.Errorf("Wrong median value: %f != %f", result, want)
+	}
+
+	want = l.Mu
+	if result := l.Mode(); result != want {
+		t.Errorf("Wrong mode value: %f != %f", result, want)
+	}
+}
+
+func TestLogisticStdDev(t *testing.T) {
+	t.Parallel()
+
+	const sq3 = 1.732050807568877293527446341505872366942805253810380 // sqrt(3)
```

Comment:
If we have this constant here, perhaps move it into the package proper as `sqrt3` and then define `StdDev` as `return l.S * (math.Pi/sqrt3)`.